modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectEntryImpl.java

```diff
@@ -84,8 +86,20 @@ public String getTitleValue() throws PortalException {
 					objectDefinition.getTitleObjectFieldId());
 
 			if (objectField != null) {
+				Map<String, Serializable> values = Stream.of(
```

Comment:
Unfortunately, Brian don't let pass code using **stream** anymore.
You could try use something like:
```
HashMapBuilder.create(
			getValues()
		).putAll(
			ObjectEntryLocalServiceUtil.getSystemValues(this)
		).build()
```